seed/ts-sdk/pagination/src/api/resources/users/client/Client.ts

```diff
@@ -532,7 +532,7 @@ export class Users {
         let _offset = request?.page != null ? request?.page : 1;
         return new core.Pageable<SeedPagination.ListUsersPaginationResponse, SeedPagination.User>({
             response: await list(request),
-            hasNextPage: (response) => (response?.data ?? []).length > 0,
+            hasNextPage: (response) => response?.hasNextPage ?? (response?.data ?? []).length > 0,
```

Comment:
core change

Reply:
Sweet, deferring to the length check is great 👍